### PR TITLE
CI: build releases for Alpine

### PR DESCRIPTION
**Pull request by miraclx**
Building for Alpine (musl) works, provide a prebuilt binary in releases.

Example: https://github.com/miraclx/atomicparsley/releases/tag/20_test_alpine_release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,16 @@ setting metadata into MPEG-4 files, in particular, iTunes-style metadata.
 * Navigate to the [latest release](https://github.com/wez/atomicparsley/releases/latest)
 * Download the `AtomicParsleyWindows.zip` file and extract `AtomicParsley.exe`
 
+### Linux
```
Comment (wez) [inline README.md:20]: Please add `(x86-x64)` here to make it clearer about the architecture

File: README.md [hunk 2]
```diff
@@ -17,6 +17,16 @@ setting metadata into MPEG-4 files, in particular, iTunes-style metadata.
...
+
+* Navigate to the [latest release](https://github.com/wez/atomicparsley/releases/latest)
+* Download the `AtomicParsleyLinux.zip` file and extract `AtomicParsley`
+
+### Alpine (musl)
```
Comment (wez) [inline README.md:25]: Let's label this as `Alpine Linux (x86-64 musl libc)`

**Review by wez — COMMENTED** [review]
Thanks for spending the time on this!
I have a couple of suggested tweaks for the docs; would you mind updating those?

I'm assuming that this PR supersedes #17 and we can close #17?

**Comment by miraclx** [comment]
Yeah, #17 is included here, you can go ahead to close that.

**Pull request merged**